objectivec/cxx_utils.mm

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "cxx_utils.h"
+
+#import <vector>
+#import <optional>
+#import <string>
+
+#import "error_utils.h"
+
+#import "ort_value_internal.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+NSString* Utils::toNSString(const std::string& str) {
```

Comment:
these can all go into `namespace utils { ... }`

also, could we name it `utils` instead?